packages/data/migrations/20181019172401-availability-status.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+var dbm
+var type
+var seed
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate
+  type = dbm.dataType
+  seed = seedLink
+}
+
+// up modifies all agents by changing the currentFreeAgent field to the availability field. No information
+// is added/lost.
+exports.up = function(db, cb) {
+  db.agents.updateMany({}, { $rename: { currentFreeAgent: 'available' } })
```

Comment:
It can't seem to find the agents collection. This exact same command runs fine when I do it myself through the mongodb shell though. I think it may be a problem with the package we're using, and the db object that it provides. 

It's strange that for the other migration though the db object seems to have the standard db methods, such as .createCollection() and .dropCollection(). 

<img width="1430" alt="screen shot 2018-10-19 at 1 56 02 pm" src="https://user-images.githubusercontent.com/33774466/47235341-bf298000-d3a6-11e8-94d8-ad87830b934b.png">